.gitignore

```diff
@@ -2,3 +2,4 @@
 cmd/tuf/tuf
 cmd/tuf-client/tuf-client
 .vscode
+.idea
```

Comment:
Please remove; editor-specific configuration [belongs in developers' `.gitignore` files](https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/).

(While you're at it, remove `.vscode` as well 🙂 )

Reply:
Took care of it. Thanks